include/private/vkd3d_common.h

```diff
@@ -118,7 +118,7 @@ static inline unsigned int vkd3d_bitmask_tzcnt32(uint32_t mask)
 {
 #ifdef _MSC_VER
     unsigned long result;
-    _BitScanForward(&result, mask) ? result : 32;
+    _BitScanForward(&result, mask);
```

Comment:
This seems unrelated?

Reply:
Unrelated but was throwing a bunch of warnings in Visual Studio Clang. Can be removed from this PR if necessary.

Reply:
Which warning? This code will break since it doesn't handle the mask == 0 case.

Reply:
`In file included from ..\..\libs\vkd3d-shader/vkd3d_shader_private.h:48:
..\..\include\private\vkd3d_common.h(121,47): warning: expression result unused [-Wunused-value]
    _BitScanForward(&result, mask) ? result : 32;`

From what I can tell, this code is not checking the mask but rather a remnant from a previous implementation. I searched the rest of the codebase and find no other instances that is like this one.

This warning is from MSVC Clang.

Reply:
Yes, this is actually just a derpy bug, fun.